nature/styleguide/typography.md

```diff
@@ -32,7 +32,33 @@ Use our font styles consistently by always making sure they reflect the informat
 * Body text and paragraphs at base sizes should not exceed 800px in width
 * We use fluid typography by applying properties for different screen sizes, "Max" and "Min" 
 
-### Headings (Nature journals)
+## Headings
+
+## Write headings in sentence case
+
+For example, “Latest reviews and analysis” not “Latest Reviews and Analysis”.
+
+Sentence case is easier for people to read.
+
+Headings should follow [our content guidelines](/content).
+
+### Make headings meaningful
+
+Use descriptive headings that convey the meaning of the section they refer to. This helps users scan and find the information they need, without having to read all of the content from top to bottom.
+For example, “Publish with us” is clearer than “Information on submission”.
+
+### Use heading levels to communicate page structure
+
+Use semantic heading levels <h1> to <h6> to show the order of importance of the content on a page.
```

Comment:
I try to avoid the term "importance" regarding heading levels. Something might be very important but detined to be (only) an `<h4>` just because of the structure. Perhaps:

_Use semantic heading levels `<h1>` to `<h6>` to indicate the relationship of sections to subsections in the page structure._

Also, use backticks for encapsulating code (\`&lt;h1>\`). I think those tags will be parsed as actual HTML if you don't mark them up as code snippets.

Reply:
Good call. How's this...

```suggestion
Use semantic heading levels `<h1>` to `<h6>` to show the relationship of sections to subsections in the page structure.
```

Reply:
👍